addon/classes/Row.js

```diff
@@ -33,6 +35,17 @@ export default class Row extends Ember.ObjectProxy.extend({
    * @type {String | Array}
    */
   classNames: null,
+
+  /**
+   * Element ID for the corresponding `{{lt-row}}`.
+   *
+   * @property elementId
+   * @type {String}
+   * @readOnly
+   */
+  elementId: computed(function() {
```

Comment:
I named this field `elementId`, so it does not shadow the `id` field of the `content` object which is most likely an Ember Data `Record`.

Still want me to rename it?